src/common/components/hosts/tableUtils.tsx

```diff
@@ -44,13 +45,14 @@ export const getSelectedNic = (nics: Interface[], currentSubnet: Address4 | Addr
 };
 
 export const hostnameColumn = (
+  t: TFunction,
   onEditHostname?: HostsTableActions['onEditHost'],
   hosts?: Host[],
   canEditHostname?: HostsTableActions['canEditHostname'],
 ): TableRow<Host> => {
   return {
     header: {
-      title: 'Hostname',
+      title: t ? t('ai:Hostname') : 'Hostname',
```

Comment:
Why can  `t` be missing? It's the first parameter and it should always be passed I think

Same question for `roleColumn` and `statusColumn` later

Reply:
It's true. At first t was optional. I've changed it